clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -165,23 +167,22 @@ class FossIdClientNewProjectTest : StringSpec({
     "Scan results can be listed" {
         service.listScanResults("", "", SCAN_CODE) shouldNotBeNull {
             checkResponse("list scan results")
-
-            data shouldNotBeNull {
+            val results = toList<FossIdScanResult>()
+            results.shouldNotBeNull {
```

Comment:
IIUC `toList()` won't ever return `null`, so `shouldNotBeNull` should be removed. 
How about...?
```
with(toList<FossIdScanResult>()) {
    size shouldBe 58
    last().localPath shouldBe "pom.xml"
}
```